gbols/fast-food
# #160450874 Endpoint to get all orders.

**What does this PR do?**
It gets all orders on the platform.

**Description of Task to be completed?**
- create endpoint to get a single order.
- create an endpoint to place a new order.
- create an endpoint to update status of an order.

**How should this be manually tested?**
- pull from this branch `ft-get-orders-160450874` into a new branch.
- run `npm install` to install all the requirements.
- endpoint should look like this `http://localhost:3000/api/v1/users/orders`.
- run `npm test` to run the test.

**What are the relevant pivotal tracker stories?**
#160450874 user should be able to get all food orders.

**Postman Screenshot.**
![screenshot from 2018-09-13 08-40-01](https://user-images.githubusercontent.com/27724644/45474917-cec40200-b732-11e8-87a2-48d411f2dbe4.png)
